packages/devkit/src/utils/project-found-in-root-path.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { projectFoundInRootPath } from './project-found-in-root-path';
+import { CreateNodesContext } from 'nx/src/project-graph/plugins';
+import { TempFs } from 'nx/src/internal-testing-utils/temp-fs';
+
+describe('projectFoundInRootPath', () => {
+  let context: CreateNodesContext;
+  let tempFs: TempFs;
+
+  beforeEach(async () => {
+    tempFs = new TempFs('workspace-tests');
+
+    process.chdir(tempFs.tempDir);
```

Comment:
I think this may be what's causing the jest failures. The tempFs gets cleaned up, so the working directory is now in a folder that doesn't exist. Further, I don't actually think it should be needed.

Can you try removing it and see if that fixes the failures? This cleanups actually pretty valuable imho, and its something I'd like to get merged. Sorry for the **_way_** too long feedback cycle here.